### PR TITLE
support dpo training on CPU (SPR)

### PR DESCRIPTION
**Pull request by lkk12014402**
## Type of Change

API not changed

## Description

support dpo training on CPU (SPR)

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]

File: intel_extension_for_transformers/neural_chat/examples/finetuning/dpo_pipeline/dpo_clm.py [hunk 1]
```diff
@@ -216,6 +216,10 @@ def find_all_linear_names(model):
 
     model_args, data_args, training_args, finetune_args = parser.parse_args_into_dataclasses()
 
+    if training_args.use_cpu:
```
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/examples/finetuning/dpo_pipeline/dpo_clm.py:219]: qlora on Xeon does not show perf benefits

Comment (lkk12014402) [reply intel_extension_for_transformers/neural_chat/examples/finetuning/dpo_pipeline/dpo_clm.py:219]: quant/dequant takes more time

Comment (hshen14) [reply intel_extension_for_transformers/neural_chat/examples/finetuning/dpo_pipeline/dpo_clm.py:219]: Then we should remove it

Comment (lkk12014402) [reply intel_extension_for_transformers/neural_chat/examples/finetuning/dpo_pipeline/dpo_clm.py:219]: done

**Pull request merged**